python/cudf/cudf/core/column/string.py

```diff
@@ -2086,6 +2086,13 @@ def to_array(self, fillna=None):
 
         return self.to_arrow().to_pandas().values
 
+    def __array__(self, dtype=None):
+        raise TypeError(
+            "Implicit conversion to a host NumPy array via __array__ is not allowed, \
+                To explicitly construct a host array, \
+                    consider using .to_array()"
```

Comment:
Lets be consistent and just indicate a gpu array isn't supported yet for strings.

Reply:
Sure, added that there is no support for gpu array for strings.